include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -5706,6 +5706,38 @@ struct OrtApi {
    */
   ORT_API2_STATUS(Graph_GetParentNode, _In_ const OrtGraph* graph, _Outptr_result_maybenull_ const OrtNode** node);
 
+  /** \brief Create a "sub-graph" from a subset of nodes in an OrtGraph.
+   *
+   * A "sub-graph" refers to a graph formed from a subset of nodes within the current OrtGraph and is not a
```

Comment:
nit: I'm not sure if differentiating `sub-graph` (with a hyphen) from `subgraph` is easy to remember. I'd be interested to hear what others think, but I would prefer one of the following:
- Call it `Graph_GetGraphView`. Still returns `OrtGraph`, but we explain it contains a subset of nodes. This is similar to the distinction between `std::string` and `std::string_view`, where the `std::string_view` can refer to a subset of characters in the original `std::string`.
- Call it `Graph_GetSubgraph` (not "SubGraph" or "sub-graph") and explain that this is not a nested control flow subgraph.